src/geff/_graph_libs/_networkx.py

```diff
@@ -135,12 +135,11 @@ def _set_property_values(
         nodes (bool, optional): If True, extract and set node properties.  If False,
             extract and set edge properties. Defaults to True.
     """
-    sparse = "missing" in prop_dict
     for idx in range(len(ids)):
         _id = ids[idx]
         val = prop_dict["values"][idx]
         # If property is sparse and missing for this node, skip setting property
-        ignore = prop_dict["missing"][idx] if sparse else False
+        ignore = prop_dict["missing"][idx] if prop_dict["missing"] is not None else False
```

Comment:
Should we use _path.MISSING in these implementations as well? (I mean, we should, but should we do it in this PR? I vote yes, we are already touching all the code)